src/fu-engine.c

```diff
@@ -3002,9 +3002,7 @@ fu_engine_udev_device_add (FuEngine *self, GUdevDevice *udev_device)
 	}
 
 	/* does the quirk specify the plugin to use */
```

Comment:
this comment should change since this is not looking via quirk anymore

Reply:
Well, it is set from a quirk typically. What about something like `can be specified using a quirk`